apps/src/code-studio/pd/application/teacher1920/Section3TeachingBackground.jsx

```diff
@@ -121,6 +121,22 @@ export default class Section3TeachingBackground extends LabeledFormComponent {
     );
   }
 
+  /**
+   * @override
+   */
+  static getDynamicallyRequiredFields(data) {
```

Comment:
Are there tests associated with the requiredness of fields on this form, that we need to update after this change?  Or can we add a test for this?

Reply:
Discussed offline, we have tests on server side but not client side. We don't have to add test for this now.